frontend/src/app/testQueue/TestCard/TestCard.tsx

```diff
@@ -76,6 +98,22 @@ export const TestCard = ({
     removeTimer(testOrder.internalId);
   };
 
+  const saveStartedAtCallback = async (startedAt: number | undefined) => {
+    const response = await updateTestOrderTimerStartedAt({
+      variables: {
+        testOrderId: testOrder.internalId,
+        startedAt: startedAt?.toString(),
+      },
+    });
+    if (!response.data) {
+      throw Error("updateTestOrderTimerStartedAt null response data");
+    }
+    if (startedAt === undefined) {
+      timer.reset(trackTimerReset);
```

Comment:
See [this comment](https://github.com/CDCgov/prime-simplereport/pull/7896/files#r1672542367) for why we pass the `trackTimerReset` function